assignments/objects.js

```diff
@@ -95,20 +95,41 @@ console.log(Antonietta.multiplyNums(3,4));
 // 2. Nest a child object in the parent object with name and age as well.  The name will be George and the age will be 50.
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
+const parent = {
+  name: 'Susan',
+  age: 70,
+  speak: function() {console.log(`Hello, my name is ${this.name}!`)},
```

Comment:
Object literals are convenient for adding methods such as these. It definitely adds to readability. Makes it easier to see which methods belong to which objects. I would advise to format your function so that it breaks over several lines though, again for the purpose of readability.